editor/src/core/model/utopia-constants.ts

```diff
@@ -5,7 +5,7 @@ export const UTOPIA_DO_NOT_TRAVERSE_KEY = 'data-utopia-do-not-traverse'
 export const UTOPIA_SCENE_ID_KEY = 'data-utopia-scene-id'
 export const UTOPIA_UID_PARENTS_KEY = 'data-utopia-parents'
 export const UTOPIA_UID_ORIGINAL_PARENTS_KEY = 'data-utopia-original-parents'
-export const UTOPIA_SCENE_PATH = 'data-utopia-scene-path'
+export const UTOPIA_INSTANCE_PATH = 'data-utopia-instance-path'
```

Comment:
should / could this be unified with the `data-paths` key?

Reply:
Good question. I'm not sure tbh. This exists purely for the DOM walker caching. I think it probably should be unified but I wouldn't want to tackle that as part of this PR

Reply:
ah okay, makes sense, thanks :)